### PR TITLE
fix: Divide by zero exception fix in item wise purchase register

### PR DESCRIPTION
**Pull request by deepeshgarg007**
Do not prepare row if stock quantity is zero
If ```d.stock_qty``` is not zero then only row for that item will be prepared

Before:
```
Traceback (most recent call last):
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/desk/query_report.py", line 96, in run
    res = frappe.get_attr(method_name)(frappe._dict(filters))
  File "/home/frappe/benches/bench-2018-11-15/apps/erpnext/erpnext/accounts/report/item_wise_purchase_register/item_wise_purchase_register.py", line 11, in execute
    return _execute(filters)
  File "/home/frappe/benches/bench-2018-11-15/apps/erpnext/erpnext/accounts/report/item_wise_purchase_register/item_wise_purchase_register.py", line 52, in _execute
    purchase_receipt, expense_account, d.stock_qty, d.stock_uom, d.base_net_amount / d.stock_qty, d.base_net_amount
ZeroDivisionError: float division by zero
```
After:
screenshot 2018-12-26 at 11 21 55 am

### CODE REVIEW

**Comment by saurabh6790** [comment]
Can you add screenshots (before/after)?

**Review by saurabh6790 — DISMISSED** [review]
Can you add screenshots (before/after)?

**Comment by deepeshgarg007** [comment]
@saurabh6790 Done.

**Review by sagarvora — APPROVED** [review]

File: erpnext/accounts/report/item_wise_purchase_register/item_wise_purchase_register.py [hunk 1]
```diff
@@ -34,34 +34,35 @@ def _execute(filters=None, additional_table_columns=None, additional_query_colum
 
 	data = []
 	for d in item_list:
-		purchase_receipt = None
-		if d.purchase_receipt:
-			purchase_receipt = d.purchase_receipt
-		elif d.po_detail:
-			purchase_receipt = ", ".join(po_pr_map.get(d.po_detail, []))
+		if d.stock_qty:
```
Comment (sagarvora) [inline erpnext/accounts/report/item_wise_purchase_register/item_wise_purchase_register.py]: you can reduce indentation to increase readability by doing something like:

```
if not d.stock_qty:
    continue
```

**Review by sagarvora — CHANGES_REQUESTED** [review]

**Review by sagarvora — APPROVED** [review]

**Pull request merged**